joblib/test/test_module.py

```diff
@@ -21,3 +22,30 @@ def test_no_start_method_side_effect_on_import():
         mp.set_start_method("loky")
     """
     check_subprocess_call([sys.executable, '-c', code])
+
+
+@pytest.mark.skipif(PY27, reason="Need python3.3+")
+def test_no_semaphore_tracker_on_import():
+    # check that importing joblib does not implicitly spawn a ressource tracker
+    # or a semaphore tracker
+    code = """if True:
+        import joblib
+        from multiprocessing import semaphore_tracker
+        # The following line would raise RuntimeError if the
+        # start_method is already set.
+        msg = "multiprocessing.semaphore_tracker has been spawned on import"
+        assert semaphore_tracker._semaphore_tracker._fd is None, msg"""
+    check_subprocess_call([sys.executable, '-c', code])
+
+
+@pytest.mark.skipif(PY27, reason="Need python3.3+")
```

Comment:
What is python3.3+ specific in this test?

Reply:
too fast on copy/paste